client/src/main/kotlin/no/nav/su/se/bakover/client/oppdrag/avstemming/DetaljBuilder.kt

```diff
@@ -22,20 +21,20 @@ class DetaljBuilder(
         }
 
     private fun mapStatus(utbetaling: Utbetaling): Detaljdata.Detaljtype = when (utbetaling.kvittertMedFeilEllerVarsel()) {
-        true -> mapUtbetalingsstatus(utbetaling.kvittering!!.utbetalingsstatus)
+        true -> mapUtbetalingsstatus((utbetaling as Utbetaling.KvittertUtbetaling).kvittering.utbetalingsstatus)
```

Comment:
when(utbetaling) {
 is Utbetaling.KvittertUtbetaling -> ..?
}

Reply:
Jepp, er vel fort mer fornuftig å gjøre det slik.